macroslib/src/java_jni/jni-include.rs

```diff
@@ -786,12 +786,12 @@ macro_rules! define_array_handling_code {
                 fn to_slice(&self) -> &[$rust_elem_type] {
                     unsafe {
                         let len: jsize = (**self.env).GetArrayLength.unwrap()(self.env, self.array);
-                        assert!((len as u64) <= (usize::max_value() as u64));
```

Comment:
This fixes "warning: usage of a legacy numeric method" caused by clippy::legacy_numeric_constants
https://rust-lang.github.io/rust-clippy/master/index.html#legacy_numeric_constants